service/internal/resources.go

```diff
@@ -1,17 +1,3 @@
-// Copyright  The OpenTelemetry Authors
```

Comment:
I think something changed with addlicense. :(

Reply:
Yes, addlicense changed to not check/add license for generated code.